incubator/kubevirt/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+The KubeVirt components have been installed.
+
+The VM API can be accessed by {{ .Values.master.ip }}:8184.
+In the case you want to create VM via kubectl command, use
+kubectl -s {{ .Values.master.ip }}:8184 -f vm.yaml
```

Comment:
Yes that is where we currently need an external IP, so that one can reach our subresources from outside the cluster.